.github/scripts/check-template-and-add-labels.ts

```diff
@@ -98,8 +99,9 @@ async function main(): Promise<void> {
     labelable.body,
   );
 
-  // If labelable's author is a bot we skip the template checks as bots don't use templates
-  if (knownBots.includes(labelable.author)) {
+  // If labelable's author is a bot we skip the rest of the script, including the template checks as bots don't use templates.
+  // Exception: For issues created the 'sentry-io' bot, we don't skip the rest of the script because there's a specific handling for those issues.
+  if (knownBots.includes(labelable.author) && labelable.author !== 'sentry-io') {
```

Comment:
nit: For early exit:
```suggestion
  if (labelable.author !== 'sentry-io' && knownBots.includes(labelable.author)) {
```

nit: `knownBots` only seems to be used internally in this file for `.includes()` calls. Would it make sense to define it as a set instead of an array? Any performance impact would be miniscule, but still since this is a CI script, every little gain should count?

Reply:
You're right, that would be cleaner